models/User.js

```diff
@@ -49,18 +49,6 @@ const UserSchema = new mongoose.Schema({
         min: -180,
         max: 180,
     },
-    latitude: {
-        type: Number,
-        required: [true],
-        min: -90,
-        max: 90,
-    },
-    longitude: {
-        type: Number,
-        required: [true],
-        min: -180,
-        max: 180,
-    },
```

Comment:
I don't think this should have been deleted.

Reply:
> I don't think this should have been deleted.

It was extra Jacob, I pull the new updates. It doubled it! So I delete one of it!

Reply:
I see. Thanks for clarifying